packages/kbn-expandable-flyout/index.ts

```diff
@@ -17,3 +17,7 @@ export type { ExpandableFlyoutApi } from './src/context';
 
 export type { ExpandableFlyoutProps } from './src';
 export type { FlyoutPanelProps, PanelPath } from './src/types';
+
+export { useUrlState } from './src/hooks/use_url_state';
+export { useLeftPanel } from './src/hooks/use_left_panel';
+export { useRightPanel } from './src/hooks/use_right_panel';
```

Comment:
any reasons for not exposing the `usePreview` hook?

Reply:
its there now:)